### ISSUE TITLE
CheckBox - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4852]
Add Accessibility information to the CheckBox page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/4675/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA (tag these with the 'bug label') or anywhere we observe that we are not meeting best practices.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

**Comment by CillianHPE** [comment]
@jcfilben 

I've made the first pass through the checkbox. Do you mind having a look to see if you agree?

**[Link to Excel](<div ccp_infra_version='3' ccp_infra_timestamp='1741963100468' ccp_infra_user_hash='2073012144' ccp_infra_copy_id='5923533f-fd13-43df-8084-46b90669180c' data-ccp-timestamp='1741963100468'><html><head><meta http-equiv=Content-Type content="text/html; charset=utf-8"><meta name=ProgId content=Excel.Sheet><meta name=Generator content="Microsoft Excel 15"><style>table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:11.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:"Aptos Narrow", sans-serif;
	mso-font-charset:0;
	text-align:general;
	vertical-align:bottom;
	border:none;
	white-space:nowrap;
	mso-rotate:0;}
.xl17
	{color:black;
	font-size:12.0pt;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	white-space:normal;}
</style></head><body link="#467886" vlink="#96607D">
Implementer needs to ensure the ARIA label is informative to the user.
--


</body></html></div>)**

**Comment by jcfilben** [comment]
CheckBox is meeting AA so no follow on issues, marking this as closed
